### DISCUSSION TITLE
Nuclei Delete Anything After # Mark

### DISCUSSION THREAD

**Discussion by 0xAwali** [projectdiscovery/nuclei#889] [Q&A] [answered]
### Nuclei Delete Anything After # e.g. Template Is

```
id: keenetic-web-login
info:
  name: Keenetic Web Login
  author: dhiyaneshDK
  severity: info
  reference: https://www.exploit-db.com/ghdb/6817
  tags: panel,keenetic

requests:
  - raw:
      - |
        GET /login#goto=%2Fdashboard HTTP/1.1
        Host: {{Hostname}}
        User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:55.0) Gecko/20100101 Firefox/55

    matchers-condition: and
    matchers:
      - type: word
        words:
          - '<title ng-bind="$root.title">Keenetic Web</title>'
      - type: status
        status:
          - 200
```

### Nuclei Will Send Request 

![Screenshot_2021-07-29_02_59_02](https://user-images.githubusercontent.com/54332136/127415635-d0522292-db8b-4ae4-ba6d-dd1cd9bd2cc9.jpg)

###  2.4.0

**Comment by geeknik** [comment]
If you add [unsafe: true](https://nuclei.projectdiscovery.io/templating-guide/protocols/http/#unsafe-http-requests) to the template does the behavior change?

**Comment by 0xAwali** [comment]
No , I'm Just Use Its Now With unsafe: true AND Work Correctly

**Comment by ehsandeep** [comment, marked as answer]
@0xAwali you only need to access `/login` to get the expected content/match, anything after`#` in URL is used on the client-side.
